themes/src/main/resources/theme/keycloak-preview/account/resources/app/content/linked-accounts-page/linked-accounts-page.component.html

```diff
@@ -1,5 +1,23 @@
 <div class="page-header">
-  <h1 id="pageTitle">{{'linkedAccountsHtmlTitle' | translate}}</h1>
+  <h1>{{'linkedAccountsHtmlTitle' | translate}}</h1>
+  <hr/>
+  <h2>Buttons for manual testing.  See JS Console for output.</h2>
+  <div class="card-pf-body row">
+      <button class="label label-primary" (click)="doGet()">List All</button>
+  </div>
+  <div class="card-pf-body row">
```

Comment:
I assume the harcoding of providers here is temporary, and it will be changed to using the rest endpoint as part of a separate issue?

Reply:
Yes, this whole file goes away when I actually implement the UI.